packages/block-library/src/navigation/block.json

```diff
@@ -50,6 +50,7 @@
 		"html": false,
 		"inserter": true,
 		"fontSize": true,
+		"__experimentalFontAppearance": true,
```

Comment:
Font appearance is not something that is normally referred to. It was a UI decision to join them. But that decision may be changed in the future and for example, we may receive feedback to separate into weight and style using a different UI like buttons or to integrate additional typography settings into weight and style. I guess it would also make sense for a block to allow tyle selection without font-weight or the contrary. To me it feels like exposing the concept of font appearance as part of the block API is a restriction. What if we individually exposed fontWeight and fontStyle in block.json, the design may still join them in a single UI control. Depending on what is enabled we would show some of the options.